package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.3.0",
   "description": "Internal administrative tools service for the GOV.UK Pay products.",
   "engines": {
-    "node": "^12.18.1"
```

Comment:
As far as I know this is only used when selecting the version of the Node buildpack when pushing to PaaS. Unfortunately it cannot always move in step with the version we're using in our Dockerfile because different versions are supported. The versions supported by PaaS can be found here: https://github.com/cloudfoundry/nodejs-buildpack/releases/